### PR TITLE
create footer with icons

### PR DESCRIPTION
**Pull request by reyk-zepper**
create the footer with icons and layout it

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -3,3 +3,21 @@
   padding: 0;
   box-sizing: border-box;
 }
+
+.navigation {
+  display: flex;
+  flex-direction: row;
```
Comment (mpagels) [inline css/style.css:9]: `flex-direction: row` is the default value, when using `display: flex;` You don't need that. :-)

File: css/style.css [hunk 2]
```diff
@@ -3,3 +3,21 @@
...
+
+.icon {
+  height: 25px;
+  width: auto;
+}
+
+.footer {
+  border: 2px solid #111;
+  margin: 10px;
+  padding: 10px;
```
Comment (mpagels) [inline css/style.css:22]: Awesome, just using one value for every side. Super!

File: index.html [hunk 3]
```diff
@@ -4,7 +4,32 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/style.css" />
     <title>Quiz App</title>
   </head>
-  <body></body>
+  <body>
+    <footer class="footer">
+      <nav class="nav-bar">
+        <ul class="navigation">
+          <li class="home-icon">
```
Comment (mpagels) [inline index.html:14]: Do you use the `home-icon` anywhere in your project? I recommend not putting code in here, when it is not used. ;-)

**Review by mpagels — APPROVED** [review]
Hey Reyk, looking very very good. Super awesome work here.

Found just some tiny little improvements. Have a look.

File: css/style.css (hunk 1 above)
Comment (reyk-zepper) [reply css/style.css:9]: thx...i forgot...

File: index.html (hunk 3 above)
Comment (reyk-zepper) [reply index.html:14]: i plan to use it, if not i will delete it

**Pull request merged**